stellargraph/core/graph.py

```diff
@@ -91,11 +89,17 @@ class StellarGraph:
 
         Gs = StellarGraph(nodes, edges)
```

Comment:
```suggestion
        Gs = StellarGraph(edges=edges)
```
and similarly for the `StellarDiGraph` call below.

Reply:
The change needs to be exactly what I said, with the keyword argument. `StellarGraph(edges)` as you adjusted it to is passing the `edges` dataframe into the `nodes` argument, and won't work correctly (I think it'll create a node for each edge, where the node features are the source and target node IDs).